### ISSUE TITLE
Integration with Leap Motion

### ISSUE THREAD

**Issue by andregm3** [cboulay/PSMoveService#131]
![ezgif com-optimize](https://cloud.githubusercontent.com/assets/4154732/17078472/607c0048-50ca-11e6-8cc8-db55161e8750.gif)

Just got my Leap Motion and it is very precise with hand tracking it could work holding the PS Move Controller, or Navigation Controller.

Maybe removing the tracking of PS Eye and using the Leap Motion track with integration of Buttons and Feedback of the PS Move Controllers.

**Comment by zelmon64** [comment]
Just to add one small thing. It would not be necessary to use PSMoveService for the buttons because the are recognised natively in Windows. I agree that the tracking isn't as good if you hold something. I was surprised that they took the tool recognition (and a few other bits) out of orion (but it was probably to improve the tracking I guess).

**Comment by andregm3** [comment]
https://www.youtube.com/watch?v=8q31ztsEYdo

hand tracking as Vive Controllers, just needed the PS Thumbstick and buttons

**Comment by cboulay** [comment]
@andregm3 When we get PSNavi support baked in then you're welcome to use PSMoveService for it, but as @zelmon64 said, there are other more-native solutions for capturing button presses on arbitrary controllers.

**Comment by andregm3** [comment]
yes, that was what i was thinking, when Navi support came out, but a shame it don't have vibration feedback, BTW did you see that RiftCat shared my video on they timeline ? i got like 3.000 views and added link to the project here, many more adepts to the PS Move !

**Comment by zelmon64** [comment]
@andregm3 You can already access the functions of a PSNavi by using the original version (and possibly the latest version) of ScpToolkit (it gets recognised as an Xbox controller).
